.circleci/config.yml

```diff
@@ -3,34 +3,30 @@ executors:
   docker-publisher:
     environment:
       IMAGE_NAME: logdna/logdna-agent
+      ICR_IMAGE_NAME: icr.io/ext/logdna-agent
     docker:
       - image: circleci/buildpack-deps:stretch
 jobs:
-  test:
-    docker:
-      - image: circleci/node:10
-    steps:
-      - checkout
-      - run:
-          name: Install Dependencies
-          command: npm install
-      - run:
-          name: Run Tests
-          command: ./node_modules/.bin/grunt test
   image-build:
     executor: docker-publisher
     steps:
       - checkout
-      - run: |
-          echo "export DATESTAMP=$(date +%Y%m%d)" > env-vars
-          echo "export MAJOR_VERSION=1" >> env-vars
-          cat env-vars >> ${BASH_ENV}
       - setup_remote_docker
       - run:
-          name: Build Docker image
+          name: Set Environment Variables
           command: |
-            echo "${DATESTAMP}"
-            docker build -t ${IMAGE_NAME}:sandbox-${DATESTAMP} .
+            DOCKER_TAG=$(wget -q https://registry.hub.docker.com/v1/repositories/logdna/logdna-agent/tags -O - | jq -r '.[] | .name' | awk '!/-/ && /\./ { print $1}' | grep '1.' | tail -1)
```

Comment:
I would still like to see a better way of managing the major version in this block.

Reply:
Updated!